marketplace/users/form.py

```diff
@@ -0,0 +1,16 @@
+from django import forms
+from .models import Seller
+
+class SellerForm(forms.ModelForm):
+    class Meta:
+        model = Seller  # or your custom User model
+        fields = ['username', 'email']
+        widgets ={
+            "username":forms.TextInput(attrs={"class": "form-control"}),
+            "email":forms.EmailInput(attrs={"class":"form-control"})
+            
+        }
```

Comment:
We need multiple inputs, and a username and password is must. This form will be different:

The form will be something like this...

```python3
from django import forms
from django.contrib.auth.forms import UserCreationForm

class RegistrationForm(UserCreationForm):
    email = forms.EmailField(required=True)

    class Meta:
        model = User  # seller model goes here
        fields = ['username', 'email', 'password1', 'password2']

    def save(self, commit=True):
        user = super(RegistrationForm, self).save(commit=False)
        user.email = self.cleaned_data['email']
        if commit:
            user.save()
        return user
```